src/app.ts

```diff
@@ -17,4 +19,16 @@ app.use("/chatgpt", chatGptRoutes);
 
 app.use(apiErrorHandler);
 
+writeFiles();
+
+const rule = new schedule.RecurrenceRule();
+rule.dayOfWeek = [new schedule.Range(1, 5)];
+rule.hour = 17;
+rule.minute = 0;
+
+const dailyUpdate = schedule.scheduleJob(rule, function () {
+  console.log("db update run");
+  writeFiles();
+});
```

Comment:
Refactoring these into a util file as a function and calling it from there would be a cleaner solution in my opinion.